src/json/config.json

```diff
@@ -69,7 +69,19 @@
       "installCommand": "tar -xf FILENAME",
       "pathCommand": "export PATH=$PWD/DIRNAME/bin:$PATH",
       "checksumCommand": "sha256sum FILENAME",
-      "osDetectionString": "Linux Mint Debian Fedora FreeBSD Gentoo Haiku Kubuntu OpenBSD Red Hat RHEL SuSE Ubuntu Xubuntu hpwOS webOS Tizen"
+      "osDetectionString": "not-to-be-detected"
+    },
```

Comment:
I'm not sure this is right (adding a new entry)?  We changed `X64_LINUXLH` to `LINUXXL` last week (so that's the Large Heap builds.

Reply:
@karianna The OpenJ9 JDK8 LH builds aren't showing up any more because they have the "old" name. Adding a separate section to cover both was my first stab at a fix to see if it worked (Hence the WIP prefix :-) ) - open to other options.

Reply:
ahhhhhhh.  Hmmm.

Reply:
I'm worried that those who follow behind us are doomed to never understand why.  But you can't add comments in a json doc.....  OK - I'll let this through if we raise another issue to document this / fix upstream

Reply:
If we modify the scripts to produce the "new" artifact name for 8 (maybe that'll just happen) then we could remove it release once the next update comes out next month - would mean we'd lose the LH build from the main adoptopenjdk website for that one (to be historic) release but certainly simpler than trying to rename anything. I can keep the originating issue open until we resolve - will add a comment now.

Reply:
(Although I'm not sure why we renamed the artifact in the first place)